### PR TITLE
Add more details to project request email

### PR DESCRIPTION
**Pull request by mathjazz**
@jotes r?

### CODE REVIEW

File: pontoon/base/views.py [hunk 1]
```diff
@@ -1257,16 +1258,29 @@ def request_projects(request, locale):
...
+        reverse('pontoon.contributor', kwargs={'email': user.email})
+    )
 
     if settings.PROJECT_MANAGERS[0] != '':
         EmailMessage(
-            'Project request for {locale} ({code})'.format(locale=locale.name, code=locale.code),
-            'Please add the following projects to {locale} ({code}):\n{projects}'.format(
-                locale=locale.name, code=locale.code, projects=projects
+            subject='Project request for {locale} ({code})'.format(locale=locale.name, code=locale.code),
+            body=u'''
```
Comment (jotes) [inline pontoon/base/views.py]: [optional] We could move the contents of this email into separate template.

File: pontoon/base/views.py [hunk 2]
```diff
@@ -1257,16 +1258,29 @@ def request_projects(request, locale):
         return HttpResponseBadRequest('Bad Request: Non-existent projects specified')
 
     projects = ''.join('- {} ({})\n'.format(p.name, p.slug) for p in project_list)
+    user = request.user
+    user_url = request.build_absolute_uri(
```
Comment (jotes) [inline pontoon/base/views.py]: I would move this into The User class.

File: pontoon/base/models.py [hunk 3]
```diff
@@ -118,12 +118,25 @@ def serialize(self):
...
 
-    return [locale.code for locale in locales]
+
+def user_gravatar_url(self, size):
+    email = hashlib.md5(self.email.lower()).hexdigest()
+    data = {'s': str(size)}
+
+    if not settings.DEBUG:
+        append = '_big' if size > 44 else ''
+        data['d'] = settings.SITE_URL + static('img/anon' + append + '.jpg')
```
Comment (jotes) [inline pontoon/base/models.py]: [nit] we could use urljoin.

**Comment by jotes** [comment]
r+

File: pontoon/base/views.py (hunk 2 above)
Comment (mathjazz) [inline pontoon/base/views.py]: Good idea!

File: pontoon/base/models.py (hunk 3 above)
Comment (mathjazz) [inline pontoon/base/models.py]: Keeping it as it is, since this was copied from the gravatar example.

File: pontoon/base/views.py (hunk 1 above)
Comment (mathjazz) [inline pontoon/base/views.py]: Keeping it in the view for now for easier access. But if we keep adding stuff, I agree it should be moved into a template.

**Pull request merged**